influxql/ast_test.go

```diff
@@ -1086,6 +1086,8 @@ func TestReduce(t *testing.T) {
 		{in: `foo(bar(2 + 3), 4)`, out: `foo(bar(5), 4)`},
 		{in: `4 / 0`, out: `0.000`},
 		{in: `1 / 2`, out: `0.500`},
+		{in: `2 % 3`, out: `2`},
+		{in: `5 % 2`, out: `1`},
```

Comment:
Done. Looks like a had a bug anyway when doing `float % integer` which is now fixed.